docker/config/app.env

```diff
@@ -0,0 +1,8 @@
+# Environment variables for local dev environment
+
+REDIS_HOST=ichnaea_redis_1
+
+# Database bits
+DB_HOST=ichnaea_mysql_1
+DB_USER=root
+DB_NAME=location
```

Comment:
It's likely there are other things we want to set in the local dev environment. For example, we probably want a different logging configuration--one that's not quiet as a mouse.

I'm pushing that off to another PR. Probably will do that and switch to Everett.